scooter/src/main.rs

```diff
@@ -22,6 +24,34 @@ struct Args {
     #[arg(index = 1)]
     directory: Option<String>,
 
+    // Text to search with
+    #[arg(short = 's', long, default_value = None)]
+    search_text: Option<String>,
+
+    // Text to replace the search text with
```

Comment:
You'll need an extra `/` for these comments to show up when running `--help`

```suggestion
    /// Text to replace the search text with
```